portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,13 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    background-color: #73C6B6;
+				margin-left: 20%;
+				margin-right: 20%;
+				border: 5px dashed gray;
+				padding: 10px 10px 10px 10px;
+    color: #1A5276;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
-}
+}
```

Comment:
Usually, files are expected to end with a new line which indicates the end of file. Perhaps you can add one here?